project_status_report/models/project.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Camptocamp (<http://www.camptocamp.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class ProjectProject(models.Model):
+    _inherit = 'project.project'
+
+    @api.depends('report_ids')
+    def _compute_report_count(self):
+        for rec in self:
+            rec.report_count = len(self.report_ids)
+
+    report_ids = fields.One2many(comodel_name='project.status.report',
+                                 inverse_name='project_id',
+                                 string='Reports')
+    report_count = fields.Integer(compute='_compute_report_count')
+
+    @api.multi
+    def action_generate_report(self):
+        """
```

Comment:
1st line should contain the 1st string and then 1 empty line and then the rest https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings ;)